ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/search-interactions/search-interactions.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import MenuAction from '../menu-action'
+import styled from '../../styles/styled-components'
+import { readableColor } from 'polished'
+import { hot } from 'react-hot-loader'
+
+type Props = {
+  id: string
+  editSearch: () => void
+  deleteSearch: (id: string) => void
+  runSearch: () => void
+}
+
+const Root = styled<{}, 'div'>('div')`
```

Comment:
If you aren't dynamically styling based on custom props, this can just be
```suggestion
const Root = styled.div`
```